lib/plugins/features/movies/index.js

```diff
@@ -28,7 +29,20 @@ exports.register = (server, options, next) => {
         query: MovieSearchValidator
       }
     }
-  }]);
+  },
+  {
+    method: 'POST',
+    path: '/movies/{id}/location',
+    config: {
+      handler: (request, reply) => {
+        reply(Controller.addLocation(parseInt(request.params.id), request.payload));
```

Comment:
feels a little weird to have logic here; I think normally we'd do something like parsing integers on some form of plugin that triggers before the controller; since you don't have that I would do this in the controller.